src/internt-vedtak/VedtakContent.tsx

```diff
@@ -25,13 +26,19 @@ export const VedtakContent: React.FC<{ vedtak: Vedtak }> = ({ vedtak }) => {
 };
 
 const Avslag: React.FC<{ vedtak: VedtakAvslag }> = ({ vedtak }) => {
+    const årsaker = vedtak.årsakerAvslag.map((årsak) => årsakAvslagTilTekst[årsak]).join(', ');
```

Comment:
Bruk gjerne `tekstEllerFeil` sånn at den kaster feil her hvis årsaken ikke er definiert. For å unngå at den viser undefined eller tom streng 😁